docs/versioned-plugins/outputs/email-v4.0.4.asciidoc

```diff
@@ -12,14 +12,16 @@ START - GENERATED VARIABLES, DO NOT EDIT!
 END - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
 
-[id="plugins-{type}-{plugin}"]
+[id="{version}-plugins-{type}s-{plugin}"]
 
 === Email output plugin {version}
 
 include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+Sends email when an output is received. Alternatively, you may include or
```

Comment:
This description is missing from the older plugins. It must be added, or the build fails.